src/cloudConsole/cloudConsole.ts

```diff
@@ -288,7 +288,8 @@ export function createCloudConsole(api: AzureAccountExtensionApi, osName: OSName
 				shellArgs,
 				env: {
 					CLOUD_CONSOLE_IPC: server.ipcHandlePath,
-				}
+				},
+				isTransient: true
```

Comment:
What does this flag do?

Reply:
It stops the terminal from persisting on reload so we don't get ugly errors like this one:

![image](https://user-images.githubusercontent.com/22795803/159773150-1169cef7-5522-4628-83d3-b804a9452210.png)